pkgs/desktops/deepin/default.nix

```diff
@@ -1,4 +1,4 @@
-{ lib, pkgs, config, libsForQt5 }:
+{ lib, pkgs, config, stdenv, libsForQt5 }:
```

Comment:
Does this really work?  The `stdenv` parameter seems unused, so what good does the override do?  I see no hash change in `deepin.dde-dock` derivation.

Reply:
> Does this really work?

No, it doesn't.  I've dropped it.  The `deepin` packageset is a tight mutually-recursive ball and it's really hard to selectively override things in there.  I will let the `deepin` maintainers deal with it.